components/butterfly/protocols/swim.proto

```diff
@@ -51,6 +51,7 @@ message Service {
   optional string pkg = 9;
   optional bytes cfg = 10;
   optional SysInfo sys = 12;
+  optional string channel = 13;
```

Comment:
@mwrock The more I thought about it after reading this conversation I don't think it does. I think you should be able to query the depot for the channel info based on the pkg identifier or ship it as a new message with the pkg identifier attached